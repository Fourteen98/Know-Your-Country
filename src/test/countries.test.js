import TestRenderer from 'react-test-renderer';
import React from 'react';
import { Provider } from 'react-redux';
import Countries from '../components/Countries'; // eslint-disable-line
import store from '../redux/configureStore';

it('Countries renders correctly', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <Countries />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
