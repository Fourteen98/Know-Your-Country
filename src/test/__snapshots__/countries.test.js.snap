// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countries renders correctly 1`] = `
<div>
  <div
    className="input-group"
    id="search"
  >
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Search Country"
      type="text"
      value=""
    />
    <div
      className="button-container"
    >
      <button
        className=""
        onClick={[Function]}
        type="button"
      >
        Search
      </button>
      <button
        className=""
        onClick={[Function]}
        type="button"
      >
        reset
      </button>
    </div>
  </div>
  <div
    className="countries-main-card"
  >
    <div
      className="countries-card"
    />
  </div>
</div>
`;
